### PR TITLE
Release 22.01.0

### PR DESCRIPTION
**Pull request by philippotto**
I also added a make-changelog.py which I adapted from https://github.com/scalableminds/webknossos-libs/blob/master/make_changelog.py to work with the unreleased/released dichotomy. Reviewing the python script is not strictly necessary in my opinion. I will battle test it in the next releases. For now, it produced the current PRs diff.

------
(Please delete unneded items, merge only when none are left open)
- [X] Updated [(unreleased) changelog](../blob/master/CHANGELOG.unreleased.md#unreleased)
- [X] Updated [(unreleased) migration guide](../blob/master/MIGRATIONS.unreleased.md#unreleased) if applicable
- [ ] Updated [documentation](../blob/master/docs) if applicable
- [ ] Adapted [wk-connect](https://github.com/scalableminds/webknossos-connect) if datastore API changes
- [ ] Adapted [wk-libs python client](https://github.com/scalableminds/webknossos-libs/tree/master/webknossos/webknossos/client) if relevant API parts change
- [ ] Needs datastore update after deployment
- [X] Ready for review

### CODE REVIEW

File: CHANGELOG.released.md [hunk 1]
```diff
@@ -7,6 +7,25 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Calendar Versioning](http://calver.org/) `0Y.0M.MICRO`.
 For upgrade instructions, please check the [migration guide](MIGRATIONS.released.md).
 
+## [22.01.0](https://github.com/scalableminds/webknossos/releases/tag/22.01.0) - 2022-01-04
+[Commits](https://github.com/scalableminds/webknossos/compare/21.11.2...22.01.0)
+
+### Added
+- Added tagging support for datasets. [#5832](https://github.com/scalableminds/webknossos/pull/5832)
```
Comment (fm3) [inline CHANGELOG.released.md:14]: This one could be a highlight maybe?

**Review by fm3 — APPROVED** [review]
LGTM, as you suggested I did not review the python script in detail.
I would suggest to move it into the tools directory if possible.

File: CHANGELOG.released.md (hunk 1 above)
Comment (philippotto) [reply CHANGELOG.released.md:14]: Very good point! immediately forgot about the highlights-section after automating the changelog update process -.- I added a print-statement to the update script to remind about this.

**Comment by philippotto** [comment]
> Note that after the move the paths in the script may have to be adapted :)

I thought so to, at first, but the paths are relative to the current working dir. So, if one executes the script from the repo's root, it works :)

**Pull request merged**